### ISSUE TITLE
How to arrange the venue

### ISSUE THREAD

**Issue by plexus** [euruject/euruject#10]
I went to last week's Angular.js meetup to snap some pics of the Bitcrowd office.

The main area is a long space, with a projector set up at the far wall. This is the view when you just walked in the door.

![imag1039](https://cloud.githubusercontent.com/assets/32212/3624359/f2f3b7bc-0e5b-11e4-8653-cff65da9c574.jpg)

![imag1038](https://cloud.githubusercontent.com/assets/32212/3624360/f2f6dd52-0e5b-11e4-8a58-25a37f98760b.jpg)

![imag1049](https://cloud.githubusercontent.com/assets/32212/3624352/f2df4e58-0e5b-11e4-9bf9-319235eb1643.jpg)

Here I'm thinking to use the projector for an "open stage", strictly for lightning talks and demos. No signup but possibly a host to moderate a bit. The other end of the room can be lounge/chill/hack space.

There is also an actual office room

![imag1044](https://cloud.githubusercontent.com/assets/32212/3624355/f2e4ac68-0e5b-11e4-9cff-43fe905f1148.jpg)

And a tiny room, not sure what would be the best use for this. Hardware hacking? Podcasting? Fill it with pillows?

![imag1043](https://cloud.githubusercontent.com/assets/32212/3624357/f2e707ec-0e5b-11e4-8c07-560e10be86bb.jpg)

There's a nice kitchen

![imag1042](https://cloud.githubusercontent.com/assets/32212/3624354/f2e3a228-0e5b-11e4-8d6e-6d5550f39e4c.jpg)

And toilets

![imag1040](https://cloud.githubusercontent.com/assets/32212/3624358/f2f0bd96-0e5b-11e4-8fd6-52b81df5e4a1.jpg)

There's this fridge we can use, but might be good to arrange a second one.

![imag1048](https://cloud.githubusercontent.com/assets/32212/3624353/f2e0411e-0e5b-11e4-9480-20b885994779.jpg)

**Comment by xamebax** [comment]
Honestly, it's hard for me to say anything constructive at this moment, I'd need more time. Just dropping this comment here so you don't feel ignored. :)

**Comment by xamebax** [comment]
Oh, I've got something! We're gonna have circa 100 people with computers. Will there be enough power outlets? Maybe we should ask everyone to bring some power/plug strips with them? Should I add this to the README?

**Comment by plexus** [comment]
Good point! Yes, please add it to the README :) I should be able to bring several. Also mention that people write their name on them.

**Comment by plexus** [comment]
Also, just thought of this: enough toilet paper :)

**Comment by xamebax** [comment]
Should people just bring their own? :P

**Comment by xamebax** [comment]
> Here I'm thinking to use the projector for an "open stage", strictly for lightning talks and demos. No signup but possibly a host to moderate a bit. 

We need host(s) then.

> The other end of the room can be lounge/chill/hack space.

Sounds cool. I'm assuming the place has enough chairs, yeah?

> There is also an actual office room

How about workshops there? There should be a list to sign up in the morning, so people can know in advance and since the space is limited. What do you think?

> And a tiny room, not sure what would be the best use for this. Hardware hacking? Podcasting? Fill it with pillows?

Not sure yet! But... how about we just hang a piece of paper with 1/2h slots and let the people themselves decide what they want/need?

Other thoughts:
- some rules on how to use the kitchen (probably some restrictions apply?) etc., printed out or enumerated during the introduction.
- is trash there recycled? If so, worth noting (yeah, I know ;)). Do we need to provide trash bags?

**Comment by plexus** [comment]
Some good ideas there! @bitboxer will have to fill us in a bit more about the house rules, e.g. trash. 

How about we set the big and small room apart for workshops or whatever, with 1 hour slots for the big room, and 1/2 hour slots for the small one? That prevents people from holding a place there the whole day.

Whoever does the "hosting" for the free stage can reiterate the rules every now and then between talks.

**Comment by bitboxer** [comment]
We hopefully have enough toilet paper :wink: . The house rules are pretty simple: don't trash the place, respect the neighbours and no smoking in the office or the atrium.

**Comment by xamebax** [comment]
> How about we set the big and small room apart for workshops or whatever, with 1 hour slots for the big room, and 1/2 hour slots for the small one? That prevents people from holding a place there the whole day.

Yeah, that's basically what I said. ;)

> The house rules are pretty simple: don't trash the place, respect the neighbours and no smoking in the office or the atrium.

Cool, thanks! I just wanted to make sure you're ok (or not) with using your kitchen supplies/appliances, so there is no _faux pas_.

**Comment by bitboxer** [comment]
We just need to clean everything afterwards :wink:

**Comment by plexus** [comment]
>  Yeah, that's basically what I said. ;)

True. I just added an explicit time slot size for the big room ;)
